utils/textmate/README.md

```diff
@@ -0,0 +1,31 @@
+# Textmate Language Definition
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+This directory contains a [TextMate](https://macromates.com/) bundle which can
+be used in various editors such as TextMate, Atom or the IntelliJ family for
+syntax highlighting of Carbon source files.
+
+If you are using TextMate, you can find documentation on how to use the bundle
+[here](https://macromates.com/manual/en/bundles#getting_more_bundles). Clone the
+repository with Git and symlink/copy the `utils/textmate` directory to any of
+the paths TextMate will search through (you can find these paths in the TextMate
+documentation above)
+
+If you are using IntelliJ or a JetBrains-family product, you can find
+documentation on how to install the bundle
+[here](https://www.jetbrains.com/help/idea/textmate.html#import-textmate-bundles).
```

Comment:
```suggestion
If you are using IntelliJ or a JetBrains-family product,
[there is documentation on how to install the bundle](https://www.jetbrains.com/help/idea/textmate.html#import-textmate-bundles).
```

This is a doc style thing, just avoiding "here" as link text.
https://developers.google.com/style/link-text

Reply:
Ah that makes a lot of sense, thanks for the link!